xml/security_spectre_meltdown_checker.xml

```diff
@@ -83,9 +83,9 @@
 <screen>&prompt.root;cd /boot
 &prompt.root;spectre-meltdown-checker.sh \
 --no-color \
---kernel <replaceable>vmlinuz-4.12.14-lp151.28.13-default</replaceable> \
---config <replaceable>config-4.12.14-lp151.28.13-default</replaceable> \
---map <replaceable>System.map-4.12.14-lp151.28.13-default</replaceable>| tee <replaceable>filename.txt</replaceable>
+--kernel <replaceable>vmlinuz-&kernel-version;-default</replaceable> \
+--config <replaceable>config-&kernel-version;-default</replaceable> \
+--map <replaceable>config-&kernel-version;-default</replaceable>| tee <replaceable>filename.txt</replaceable>
```

Comment:
system map was turned into config?

Reply:
Thanks - good point, I have to check that one again...

Reply:
It was indeed a typo, thanks for spotting! Fixed with https://github.com/SUSE/doc-sle/pull/1671/commits/22e3140a87a069d22f35295932e7014b1640075a